cirq-core/cirq/ops/gate_operation.py

```diff
@@ -229,14 +229,14 @@ def _is_measurement_(self) -> Optional[bool]:
         # Let the protocol handle the fallback.
         return NotImplemented
 
-    def _measurement_key_(self) -> Optional[str]:
-        getter = getattr(self.gate, '_measurement_key_', None)
+    def _measurement_key_str_(self) -> Optional[str]:
+        getter = getattr(self.gate, '_measurement_key_str_', None)
```

Comment:
For simplicity, these should invoke `protocols.measurement_key[s]_str(self.gate)` instead - see comment on the protocol for details.

Reply:
Replied on the other comment